src/guide/user/demo.md

```
@@ -0,0 +1,91 @@
+<!---
+How to write block comments [you can do anything you want in here]
+--->
+
+[comment]: # (How to write line comments [must not contain brackets])
```

markdownlint-cli2 (0.23.1)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "comment"

(MD053, link-image-reference-definitions)